wagtail/wagtailadmin/templates/wagtailadmin/account/account.html

```diff
@@ -10,13 +10,13 @@
         <ul class="listing">
             <li class="row row-flush">
                 <div class="col6">
-                    <a href="https://gravatar.com/emails/" class="button button-primary">{% trans "Set gravatar" %}</a>
+                    <a href="{% url 'wagtailadmin_account_change_avatar' %}" class="button button-primary">{% trans "Set avatar" %}</a>
                 </div>
 
                 <small class="col6">
-                    {% trans "Your avatar image is provided by Gravatar and is connected to your email address. With a Gravatar account you can set an avatar for any number of other email addresses you use." %}
+                    {% trans "Change your avatar" %}
```

Comment:
Is the term 'avatar' widely understood by non-technical users? Github and Instagram both use 'profile picture'. I suggest we change 'avatar' to 'profile picture' in all user-facing (English) text.